diesel/src/pg/connection/row.rs

```diff
@@ -9,11 +9,17 @@ use std::rc::Rc;
 pub struct PgRow {
     db_result: Rc<PgResult>,
     row_idx: usize,
+    field_count: usize,
```

Comment:
It will be valuable for other contributors if you copy an explanation for existence of this field from the ticket into field doc